src/test/regress/expected/create_distributed_table_concurrently.out

```diff
@@ -57,6 +57,51 @@ ERROR:  cannot colocate tables nocolo and test
 DETAIL:  Distribution column types don't match for nocolo and test.
 select create_distributed_table_concurrently('test','key', colocate_with := 'noexists');
 ERROR:  relation "noexists" does not exist
+select citus_set_node_property('localhost', :worker_1_port, 'shouldhaveshards', false) from pg_dist_node;
```

Comment:
you can remove the ` from pg_dist_node` part, since we're passing in all the information directly

Reply:
well, then my feedback changes to: This thing returns 5 rows, I'm quite not sure what we're updating here. Can we just update the 2 worker nodes? :)